README.md

```diff
@@ -88,9 +89,56 @@ to tools provided by the `reset`. The `obs_msgs` returned from the environment a
 general messages that are observations. The `reward` is a scalar value. The `done` is a boolean value. The `truncated`
 is a boolean value.
 
-Let's see a complete example for building an environment.
+## Functional Environments
 
-### Environment subclass and state
+The easiest way to create an environment is using the functional interface, which just uses functions and decorators to define environments. First, let's define what the environment looks like by defining its `start` function:
+
+```py
+from aviary import fenv
+
+
+@fenv.start()
+def my_env(topic):
```

Comment:
```suggestion
def my_env(topic: str):
```

Reply:
Intentionally left it that way, since the other ones matter for LLM formatting, whereas those are optional.

Reply:
Okay I see, like what you're thinking, the presence of a type hint indicates its use by our framework

We could add a mention of that to the `README`, but can YAGNI that